### PR TITLE
Add and enforce style guide for javascript

### PR DESCRIPTION
**Pull request by rolandgeider**
See #317

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,32 +21,47 @@ python:
...
 install:
-  - sh -c "if [ '$DB' = 'postgresql' ]; then pip install psycopg2; fi"
+  # Update nvm and set wanted Node version.
+  # We update nvm using the script method instead of git, which is selected
+  # automatically, as git won't work because the $HOME/.nvm is not a git
+  # repository and the directory is not empty.
+  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.4/install.sh | METHOD=script bash
+  - . $HOME/.nvm/nvm.sh
+  - nvm install $TRAVIS_NODE_VERSION
+  - nvm use $TRAVIS_NODE_VERSION
```
Comment (rolandgeider) [inline .travis.yml]: Is it necessary to manually install nvm? Can't we specify a version in .travis [like here]
(https://docs.travis-ci.com/user/languages/javascript-with-nodejs)?

**Comment by rolandgeider** [comment]
Looks great, that was a lot of work!

We still need some installation instructions for the development stuff for the node stuff, so it's possible to test this on the local machine, but after this I think we can merge the PR.

File: .travis.yml (hunk 1 above)
Comment (petervanderdoes) [inline .travis.yml]: You can't tell Travis to use a different version unfortunately.

The nvm version installed on Travis is very old, 0.23.3 released on Feb 2, 2015. This part updates nvm to version 0.31.4.

Comment (rolandgeider) [inline .travis.yml]: Ahhh, al right then

**Comment by petervanderdoes** [comment]
I created and changed some documentation.
Should I move the branch to the wger-project repo? Would that make it easier for you?

**Comment by rolandgeider** [comment]
Thanks, no need to move it, just merged it

**Pull request merged**